docker/Dockerfile.xenial_debug

```diff
@@ -26,3 +27,11 @@ RUN apt-get update && \
    && apt-get upgrade -y \
    && apt-get clean \
    && rm -rf  /var/log/*log /var/lib/apt/lists/* /var/log/apt/* /var/lib/dpkg/*-old /var/cache/debconf/*-old
+
+# upgrade libstdc++6
+# hadolint ignore=DL3008
+RUN add-apt-repository ppa:ubuntu-toolchain-r/test \
+ && apt-get update \
```

Comment:
can we merge this all with the install block above?

Reply:
Yes. I don't have strong preference.

Reply:
Please don't, because then, in theory, `ppa:ubuntu-toolchain-r/test` could be used for other packages.

Reply:
i.e. do it after `apt-get upgrade`